converter/rst/list.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+FOUR_SPACES = '    '
+
+
+class List(object):
+    def __init__(self, source_string):
+        self.str = source_string
+        self._list_re = re.compile(r"""\n(?P<tabs>[\t]+)(?P<marker>[-|]?)?""")
+
+    @staticmethod
+    def _list(matchobj):
+        tabs = matchobj.group('tabs')
+        marker = matchobj.group('marker')
+        indent_size = 0
+        if tabs:
+            indent_size = len(tabs)
+        if marker and marker == "|":
+            indent_size += 1
+        if indent_size >= 1:
+            indent_size = indent_size - 1
+        content = '* '
+        content = indent_size * FOUR_SPACES + content
```

Comment:
`content = ' ' * indent_size * TAB_SIZE + content`
also, is it expected - if `indent_size == 0` - no indents?

Reply:
yes